src/main/java/org/acm/auth/commands/ReactionRolesCommand.java

```diff
@@ -0,0 +1,58 @@
+package org.acm.auth.commands;
+
+import net.dv8tion.jda.api.EmbedBuilder;
+import net.dv8tion.jda.api.Permission;
+import net.dv8tion.jda.api.entities.Message;
+import net.dv8tion.jda.api.entities.Role;
+import net.dv8tion.jda.api.events.message.MessageReceivedEvent;
+import org.acm.auth.utils.ReactionRoles;
+
+import java.time.Duration;
+
+public class ReactionRolesCommand extends Command {
+    private static final Permission[] EMPTY_PERMS = {};
+
+    public ReactionRolesCommand() {
+        super("rr", "Adds emoji to a msg for giving a role to users", false, false, new String[]{}, 1, 3, "", EMPTY_PERMS, EMPTY_PERMS);
+    }
+
+    @Override
+    public void invoke(MessageReceivedEvent event, String[] args) {
+        //-rr help
+        //or
+        //-rr <msg-id> <role> <reaction>
+
+        String messageId = args[0];
+        if (messageId.equals("help")) {
```

Comment:
Let's ignore case sensitivity to be more lenient
```suggestion
        if (messageId.equalsIgnoreCase("help")) {
```